### DISCUSSION TITLE
TrayIcon Howto:  Show Notification Balloon

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#6726] [Q&A] [answered]
Hello,

I want to use the new TrayIcon feature.

How can I show a balloon text, or a custom baaloon like in other UI frameworks?
E.g. https://docs.microsoft.com/de-de/dotnet/api/system.windows.forms.notifyicon.showballoontip?view=windowsdesktop-5.0
Or http://www.hardcodet.net/wpf-notifyicon/comment-page-5

With best regards.

**Comment by danwalmsley** [comment, marked as answer]
@FG-rgb in this first version of TrayIcon we are only supporting Menus.

**Reply by ApaulMyLittleAirport** [reply]
Is the Show Notification Balloon included in the next step?
